src/components/elements/EventsForMonth.jsx

```diff
@@ -15,36 +16,67 @@ const styleClasses = {
   `
 }
 
+function RenderEventType(data) {
+  const eventStyles = `inline-block mr-1 pl-2 pr-2 rounded bg-event-tags-${data.data} text-white`
+
+  return <span className={eventStyles}>{data.data}</span>
+}
+
 function formatEventTags(tags) {
   if (tags === null || tags === undefined) return [];
 
   return tags.trim().split(',').filter(tag => tag !== '');
 }
 
+function DateRange(event) {
+  const dateStart = event.start.split(' ')[0]
+  const dateEnd = event.end.split(' ')[0]
+
+  if(dateStart === dateEnd) {
+    return <p className="truncate">{event.start.slice(0, -6) + ' - ' + event.end.substring(event.end.indexOf(' ') + 1)}</p>
```

Comment:
Let's take just the date formatting portion, and turn it into a util and write a simple test for it.